plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -68,4 +68,10 @@ private MLCommonsSettings() {}
         .simpleString("plugins.ml_commons.exclude_nodes._name", Setting.Property.NodeScope, Setting.Property.Dynamic);
     public static final Setting<Boolean> ML_COMMONS_ALLOW_CUSTOM_DEPLOYMENT_PLAN = Setting
         .boolSetting("plugins.ml_commons.allow_custom_deployment_plan", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
+
+    public static final Setting<Boolean> ML_COMMONS_MODEL_AUTO_REDEPLOY_ENABLE = Setting
+        .boolSetting("plugins.ml_commons.model.autoredeploy.enable", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Comment:
```suggestion
        .boolSetting("plugins.ml_commons.model.auto_redeploy.enable", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Reply:
Should we disable auto redeploy by default?

Reply:
have reset to true by default

Reply:
sorry,yaliang
If it is set to true by default,the ci process on github will report error,you can see these 2 links:
```
https://github.com/opensearch-project/ml-commons/actions/runs/4583717456/jobs/8098013040?pr=836
https://github.com/opensearch-project/ml-commons/actions/runs/4583717456/jobs/8098013069?pr=836
```
So, I changed to false by default again~